### ISSUE TITLE
Restore when navigating away?

### ISSUE THREAD

**Issue by drventure** [bwydoogh/obsidian-force-view-mode-of-note#25]
I love this plugin, but I've noticed one minor thing.

I have it set to open my HOME note in reading mode and that works perfect, but it also appears to turn ON "source" mode, such that when I navigate back to an already opened note that was opened in edit mode with source turned off, the source mode stays on, which means I'm no longer in edit preview mode, which is a bit of a drag, because I then have to turn off SOURCE mode in order to see the live preview.

**Comment by bwydoogh** [comment]
> HOME note in reading mode

So I suppose via `obsidianUIMode: preview`? 

> but it also appears to turn ON "source" mode

However, it does not seem to me - at least not when I test this now (and I use my own plug-in, by the way) - that the "edit mode" changes to "source". With me, it remains "live preview".

What is your "Default editing mode" in Obsidian? I suppose "Live Preview"?

**Comment by bwydoogh** [comment]
Hi @drventure 

I am closing this ticket (no more feedback). Feel free to provide further information, and we'll revisit this.

**Comment by drventure** [comment]
Hi

My apologies for not getting back sooner.

I played with the settings and config a little more and realized that I needed to turn ON the options to IGNORE existing windows and Ignore when not in frontmatter.

With those 2 on, when I open a NEW window with a note, the configured view mode switches properly but then navigating to other notes doesn't change the source mode.

So it looks like everything's gold!

Thanks again!
